jenkins/e2e-image/e2e-runner.sh

```diff
@@ -84,6 +84,54 @@ function running_in_docker() {
     grep -q docker /proc/self/cgroup
 }
 
+### FUNCTIONS TO SUPPORT get-kube.sh ###
+
+# Moves local tarballs into a local path suitable for get-kube.sh, and sets
+# KUBERNETES_RELEASE and KUBERNETES_RELEASE_URL accordingly.
+function set_release_vars_from_local_gcs_stage() {
```

Comment:
We started added unit tests for non-trivial test-infra functionality. Which of these methods are interesting to unit test? What is the best way to add this unit test coverage?